src/test/java/org/broadinstitute/consent/http/authentication/OAuthAuthenticatorTest.java

```diff
@@ -56,10 +66,9 @@ void testAuthenticateGetUserInfoSuccess() {
     headerMap.put(ClaimsCache.OAUTH2_CLAIM_email, List.of("email"));
     headerMap.put(ClaimsCache.OAUTH2_CLAIM_name, List.of("name"));
     headerCache.loadCache(bearerToken, headerMap);
-    oAuthAuthenticator = new OAuthAuthenticator(samService);
+
     Optional<AuthUser> authUser = oAuthAuthenticator.authenticate(bearerToken);
-    assertTrue(authUser.isPresent());
-    assertNotNull(authUser.get().getEmail());
+    assertNotNull(authUser.orElseThrow().getEmail());
     assertNotNull(authUser.get().getAuthToken());
```

Comment:
To avoid calling `get()` again you could use `orElseThrow()` earlier
```
    AuthUser authUser = oAuthAuthenticator.authenticate(bearerToken).orElseThrow();
    assertNotNull(authUser.getEmail());
    assertNotNull(authUser.getAuthToken());
```